### DISCUSSION TITLE
Translating Docusaurus Website Manually

### DISCUSSION THREAD

**Discussion by RumeelHussainbnb** [facebook/docusaurus#8560] [Q&A]
Hi,
I have a documentation website built in Docusuarus in English. However, I also want translation of the same in multiple other languages as well. I have currently manually translated the entire documentation into Korean. The problem is now I am unable to use them with my website. Please guide me on what are the possible ways to using the manual translations with the current website.

**Comment by slorber** [comment]
It is impossible to help you because you don't explain why it doesn't work. 

"I am unable" does not provide any helpful information.

I you are looking for help, join our Discord server, some members will help you figure things out if you can share more details.
The best way is to create a repro showing your problem.

**Reply by RumeelHussainbnb** [reply]
Hi,
I have already tried Discord, but they referred me to Github instead. The official docs do not state any indication of how to use manually translated pages with the docusaurus website for localization. That's why I need a little help on how to place these translations to be able to use them

**Reply by slorber** [reply]
It is still very unclear what you are trying to do. This repo is not a chat so please make an effort to present your case in great details. Including which content plugins you are using, which files you want to translate exactly, and eventually a small repro. Help us help you, or you won't get any answer.

---

Still going to give it a try.

> how to use manually translated pages with the docusaurus website for localization. 

I'm assuming you want to translate using the Git/file-system workflow (not using Crowdin):
https://docusaurus.io/docs/next/i18n/git

I'm assuming you use the pages plugin:
https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-pages#i18n

React pages can be translated using the `<Translate>` API: https://docusaurus.io/docs/next/i18n/tutorial#translate-your-react-code

To me the docs is here. If this doesn't help you really have to make your question much more specific and detailed.
